docs/installation.md

```
@@ -51,94 +81,224 @@ ingress:
+### Client Environment Variables
+
+| Variable                    | Description                                                                                       | Required |
+|-----------------------------|---------------------------------------------------------------------------------------------------|----------|
+| `ARGO_WATCHER_URL`          | URL of the Argo Watcher server instance                                                          | Yes      |
+| `ARGO_APP`                  | Name of the Argo CD application to monitor                                                       | Yes      |
+| `COMMIT_AUTHOR`             | Person who triggered the deployment                                                              | Yes      |
+| `PROJECT_NAME`              | Identifier for the business project (not the Argo CD project)                                    | Yes      |
+| `IMAGES`                    | Comma-separated list of image names expected to contain the specified tag                        | Yes      |
+| `IMAGE_TAG`                 | Image tag expected to be deployed                                                                | Yes      |
+| `ARGO_WATCHER_DEPLOY_TOKEN` | Deploy token for Git image override (required when using the built-in GitOps updater)            | No       |
+| `BEARER_TOKEN`              | JWT token for authentication (prefix with `Bearer `, e.g. `Bearer <token>`)                     | No       |
```

markdownlint-cli2 (0.21.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)